bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/discovery/VeluxDiscoveryService.java

```diff
@@ -303,4 +309,21 @@ private void discoverBridges() {
             thingDiscovered(result);
         }
     }
+
+    @Override
+    protected void startBackgroundDiscovery() {
+        logger.trace("startBackgroundDiscovery() called.");
+        if (backgroundTask == null || backgroundTask.isCancelled()) {
+            this.backgroundTask = scheduler.scheduleWithFixedDelay(this::startScan, 10, 600, TimeUnit.SECONDS);
+        }
+    }
+
+    @Override
+    protected void stopBackgroundDiscovery() {
+        logger.trace("stopBackgroundDiscovery() called.");
+        ScheduledFuture<?> task = this.backgroundTask;
+        if (task != null && !task.isCancelled()) {
```

Comment:
No harm in cancelling something already cancelled.
```suggestion
        if (task != null) {
```

Reply:
Ok. Done.